### ISSUE TITLE
Poor performance on custom dataset

### ISSUE THREAD

**Issue by blackmrb** [fudan-zvg/PVG#25]
Thanks for the great work~
I use 3 camera(front_far, front_left front_right), 50 frames, panda 128 lidar on my own dataset, the other config is same as default cofig.
# the train and test psnr
{"split": "train", "iteration": 30000, "psnr": 31.93791029746072, "ssim": 0.9148447503123367, "lpips": 0.2528826928975289}
{"split": "test", "iteration": 30000, "psnr": 23.408180978563095, "ssim": 0.7495270156198077, "lpips": 0.3991368371579382}
# the render result of test data
![image](https://github.com/fudan-zvg/PVG/assets/165770555/710b8640-c3f9-41f7-a639-fd7b10e201da)
![image](https://github.com/fudan-zvg/PVG/assets/165770555/812da6b9-eca3-49cc-9d19-6128cbe225c6)
![image](https://github.com/fudan-zvg/PVG/assets/165770555/20f4bcdf-1012-4671-9b9e-80e60cd3c70a)

The reder image seem poor, and I found that the first few frames were rendered okay, but the later ones were much worse. I changed `fix_radius` to 10.0 and 15.0, but it seems not work. Could you please give me some advise? What parameters need to be adjusted?

**Comment by CuriousCat-7** [comment]
do you have any lidar pointcloud in your side view?

**Comment by Fumore** [comment]
Hi, it seems that the images are not captured by a pinhole camera.  Our method may fail to reconstruct the scene if your images are not well rectified.

**Comment by blackmrb** [comment]
> Hi, it seems that the images are not captured by a pinhole camera. Our method may fail to reconstruct the scene if your images are not well rectified.
![image](https://github.com/fudan-zvg/PVG/assets/165770555/4a23e54d-4289-44dd-9ecf-2988586c6790)

Thanks, maybe is camera model reason. I just use front camera which is pinhole, the result seems normal.

**I test another static scene only using front camera(50 frames), the test data set render is OK, but when add a 0.5m offset on the raw pose, the rendered image is very poor.** 
# test render image (test_30000_render)
007
![image](https://github.com/fudan-zvg/PVG/assets/165770555/24e6cd31-d9c0-40ec-8aa6-ecd22f1ea420)
027
![image](https://github.com/fudan-zvg/PVG/assets/165770555/53978088-74cb-4d3c-b8ad-4288ad528a82)
047
![image](https://github.com/fudan-zvg/PVG/assets/165770555/be926a12-ed89-4789-a609-eb7ba4af34ac)

# nvs render image,  横向y +0.5m
007
![image](https://github.com/fudan-zvg/PVG/assets/165770555/dbc79b4e-264d-433e-8f5d-8ea1a3e47a93)
027
![image](https://github.com/fudan-zvg/PVG/assets/165770555/f3f64e6f-dd05-4184-b80e-948aec485a27)
047
![image](https://github.com/fudan-zvg/PVG/assets/165770555/843524d1-408a-4ec5-8389-daa774473505)

## My impletion is below, is something wrong?
### Step 1: Save the transform, cam_infos, and c2ws.
**cam_infos_copy = copy.deepcopy(cam_infos)
c2ws_copy = copy.deepcopy(c2ws)**
c2ws, **transform**, scale_factor = transform_poses_pca(c2ws, fix_radius=args.fix_radius)

### Step 2: When generate new pose, the cam_infos is `c2ws_copy `, c2ws is `c2ws_copy `, I add 0.5m translate to raw pose, then do the same transform as `transform_poses_pca()`
```
def cameraList_from_camInfos(cam_infos, resolution_scale, args, transform=None, c2ws=None):
    camera_list = []
    for id, c in enumerate(tqdm(cam_infos)):
        if transform is not None and c2ws is not None:
            c2w_raw = c2ws[id]
            c2w_raw[1, -1] += 0.5  # y axis
            c2w = transform @ c2w_raw # same transform
            w2c = np.linalg.inv(c2w)
            c.R[:] = np.transpose(w2c[:3, :3])  # R is stored transposed due to 'glm' in CUDA code
            c.T[:] = w2c[:3, 3]
        camera_list.append(loadCam(args, id, c, resolution_scale))

    return camera_list
```

**Comment by blackmrb** [comment]
> do you have any lidar pointcloud in your side view?

no lidar in side view in the first scene, there is a front m1 lidar. @CuriousCat-7

**Comment by CuriousCat-7** [comment]
> > do you have any lidar pointcloud in your side view?
> 
> no lidar in side view in the first scene, there is a front m1 lidar. @CuriousCat-7

Yeah, as far as I know, their method should perform unwell on dynamic objects when there is no lidar supervision. This is kind of a general conclusion with gaussian splatting based method. Try mono-depth's pesudo-pointcloud may alleviate this problem.
